diff -uprN xwt/Xwt.WPF/Xwt.WPF.csproj xwt.new/Xwt.WPF/Xwt.WPF.csproj
--- xwt/Xwt.WPF/Xwt.WPF.csproj	2018-11-10 01:45:14.316442850 +0300
+++ xwt.new/Xwt.WPF/Xwt.WPF.csproj	2018-11-10 02:44:05.616968622 +0300
@@ -38,10 +38,18 @@
     <AssemblyOriginatorKeyFile>..\xwt.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="PresentationCore" />
-    <Reference Include="PresentationFramework" />
-    <Reference Include="PresentationFramework.Aero" />
-    <Reference Include="System" />
+    <Reference Include="PresentationCore">
+      <HintPath>..\..\PresentationCore.dll</HintPath>
+    </Reference>
+    <Reference Include="PresentationFramework">
+      <HintPath>..\..\PresentationFramework.dll</HintPath>
+    </Reference>
+    <Reference Include="PresentationFramework.Aero">
+      <HintPath>..\..\PresentationFramework.Aero.dll</HintPath>
+    </Reference>
+    <Reference Include="System">
+      <HintPath>..\..\System.dll</HintPath>
+    </Reference>
     <Reference Include="System.Core" />
     <Reference Include="System.Drawing" />
     <Reference Include="System.Windows.Forms" />
@@ -49,7 +57,9 @@
     <Reference Include="System.Xml" />
     <Reference Include="UIAutomationProvider" />
     <Reference Include="UIAutomationTypes" />
-    <Reference Include="WindowsBase" />
+    <Reference Include="WindowsBase">
+      <HintPath>..\..\WindowsBase.dll</HintPath>
+    </Reference>
   </ItemGroup>
   <ItemGroup>
     <Compile Include="AssemblyInfo.cs" />
